### DISCUSSION TITLE
Nuxt server with Certbot certificate.

### DISCUSSION THREAD

**Discussion by lightningnetworkstores** [nuxt/nuxt#9359] [Questions (Nuxt 2)]
In the documentation, I found an example of how to provide [SSL certificate files](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-server). However when I replace the filenames with the ones generated by certbot (like `/etc/letsencrypt/live/my-domain.com/chain.pem`), it doesn't seem to work. 

Is there an example of how to set up Nuxt with Certbot files? Thanks.

**Comment by StanleyMasinde** [comment]
Hi @lightningnetworkstores. You need to be root to use the certbot certs You can copy them the root of your application and use it from there.

 Even though you can do this, That is not the recommended way to do it. Please use a web server like Nginx as a reverse proxy fro your application. It offers features like load balancing and caching if you configure it correctly

**Reply by lightningnetworkstores** [reply]
It's not working even if I copy the files to a local directory... I know I can use Nginx but it's much easier to use Cloudflare, for small projects Nginx is not justified IMO.
